h1st/model/fuzzy/fuzzy_logic_rules.py

```diff
@@ -0,0 +1,101 @@
+import logging
+from typing import NoReturn
+
+import skfuzzy
+from skfuzzy import control as skctrl
+from skfuzzy.control.term import Term
+
+from h1st.model.fuzzy.enums import FuzzyMembership as fm
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class FuzzyLogicRules:
+    def __init__(self) -> None:
+        self.membership = {
```

Comment:
Again, is this a variable that needs to be dynamically changed, or is it a fixed constant?

Reply:
it is fixed constant.